src/bp/core/security/auth-service.ts

```diff
@@ -92,7 +92,9 @@ export class AuthService {
       return strategy && this._getStrategyConfig(strategy, strategyName)
     })
 
-    return { strategies: strategies.filter(Boolean), isFirstUser: await this.isFirstUser() }
+    const hideDefault = config.authStrategies.default && config.authStrategies.default.hide
+
+    return { strategies: strategies.filter(Boolean), isFirstUser: await this.isFirstUser(), hideDefault }
```

Comment:
What if a user want's to hide LDap for instance and show default ? 

You can make this logic generic and use the strategies.filter to filter out hidden strategies. What do you think

Reply:
@EFF thanks for feedback, yes I think that would make sense 👍 
Will implement that asap :)